feature-libs/checkout/base/root/model/checkout-step.model.ts

```diff
@@ -1,4 +1,4 @@
-export enum CheckoutStepType {
+export const enum CheckoutStepType {
```

Comment:
removed paymentType here and fixed the type augmentation in b2b 😉 
https://github.com/SAP/spartacus/pull/14174/commits/075e9c7e66271621b932f059a45de69aa07e07b5

Reply:
wellllllll

build keep failing because of this lol.... I'll fix the types tomorrow 🙈

Reply:
🙏🏻